docker-compose.prod.yaml

```
@@ -0,0 +1,9 @@
+version: "3.8"
+services:
+  web:
+    build: .
+    image: ghcr.io/cfc-servers/cfc_heartbeat_server:latest.
+    ports:
+      - "8123:8123"
```

Should probably make this:
```suggestion
      - "$PORT:8123"
```
so we can set the port per-service in the `.env`